src/javascript/JContent/ContentRoute/ContentLayout/EmptyDropZone/EmptyDropZone.jsx

```diff
@@ -37,7 +39,19 @@ const styles = theme => ({
 });
 
 const EmptyDropZone = ({component: Component, t, classes, mode}) => {
-    if (mode === JContentConstants.mode.MEDIA) {
+    const currentState = useSelector(state => ({site: state.site, language: state.language}));
+    const permissions = useNodeChecks({
+        path: `/sites/${currentState.site}`,
+        language: currentState.language
+    }, {
+        requiredSitePermission: ['uploadFilesAction']
```

Comment:
Maybe we could put this in .constants file so that we don't duplicate it in UploadTransformComponent?

Reply:
Sure ya make sense let me consolidate that into 1 variable into constants